src/Player.cpp

```diff
@@ -67,3 +67,28 @@ void Player::ResetHit()
 {
 	gequacked = false;
 }
+
+void Player::MoveAccToAxis(int x, int y, const float &dt)
```

Comment:
I have a couple of issues with this function. But inistead of correcting it, I would ask you to remove it for now and just use _duckySprite.move(vel * dt, 0.f);_ directly for now. I will want to create a separate move function for the player very soon, but let's have it as a separate issue/task. Also it will make it easier to later resolve conflict with #32 and I have never resolved conflicts for git before so let's not complicate it further for ourselves :D